src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -147,40 +147,20 @@ def build_action_text(identity: RpcIdentity, action: MessageAction) -> str | Non
     return f"*Issue {status} by <@{identity.external_id}>*"
 
 
-def build_tag_fields(
-    event_for_tags: Any, tags: set[str] | None = None
-) -> Sequence[Mapping[str, str | bool]]:
-    fields = []
-    if tags:
-        event_tags = event_for_tags.tags if event_for_tags else []
-        for key, value in event_tags:
-            std_key = tagstore.backend.get_standardized_key(key)
-            if std_key not in tags:
-                continue
-
-            labeled_value = tagstore.backend.get_tag_value_label(key, value)
-            fields.append(
-                {
-                    "title": std_key.encode("utf-8"),
-                    "value": labeled_value.encode("utf-8"),
-                    "short": True,
-                }
-            )
-    return fields
-
-
-def format_release_tag(value: str, event: GroupEvent | Group):
+def format_release_tag(value: str, event: GroupEvent | None) -> str:
     """Format the release tag using the short version and make it a link"""
+    if not event:
+        return ""
```

Comment:
Would it be better if we return `None` here instead of `""`? Otherwise the caller will have to perform falsey checks to validate whether this was successful or not.

Reply:
this will show up as a release tag like the below, i believe `None` gets rendered as "null" when dumped into json. i'm not sure which is preferable tbh

<img width="423" alt="Screenshot 2024-07-30 at 15 06 55" src="https://github.com/user-attachments/assets/a986d1c8-477a-43b2-8886-ace4f45eb4aa">